### PR TITLE
Create 2024-01-15.md

### PR DESCRIPTION
**Pull request by Cofson**

### CODE REVIEW

File: content/nomos/updates/2024-01-15.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+- [On-going discussion](https://www.notion.so/Mixnet-Specification-807b624444a54a4b88afa1cc80e100c2?d=6958059d06c14f6e847acd9d2eb8dfbb&pvs=4#96f2967aafca4269bc183e0dea59d193) about mix node identity is in progress, there are some concerns to be addressed.
```
Comment (alvatar) [inline content/nomos/updates/2024-01-15.md]: I would remove this one (unless the the discussion is summarized here)

File: content/nomos/updates/2024-01-15.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+- Three quarters of mixnet python specification code has been done; since it has been decided to move [Sphinx out of the mixnet specs](https://github.com/logos-co/nomos-specs/pull/46)(see comments) - it will be moved to a new repo in order to be utilized properly; the [basic structure and topology construction](https://github.com/logos-co/nomos-specs/pull/44) and [Sphinx packet builder](https://github.com/logos-co/nomos-specs/pull/47) have also been added.
+- Put effort into the research of [QUIC and QUIC connections](https://www.notion.so/QUIC-and-QUIC-connections-management-bd7974e3a9914b8d819fd50e6f6a6410) - what is available and what is the difference from the TCP connections
```
Comment (alvatar) [inline content/nomos/updates/2024-01-15.md]: Rather say "Research of..."

File: content/nomos/updates/2024-01-15.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+- We were able to answer most of the open questions (from previous weekly - remaining is analytical grounds for fast convergence) - How big should `T` be (# of slots in an epoch), How small should `h` be , How fast does this converge.
+- We have a general solution to the [total stake inference problem](https://www.notion.so/Lottery-Function-65f5ed5522b64c36b625652023318d88?pvs=4#7978b9a3f3964c3b9751b2644875506e) - based on this document we have a good understanding of Accuracy, Convergence Rate and Stability.
+
+### `development`
+
+- The Cryptarchia [development plan](https://www.notion.so/Crypsinous-planning-f18ea790291141bea19b496c1f59b28c) initially stated is still valid and has been updated. We will have the first milestone defined soon as well.
+- Writing of the Cryptarchia specification is well underway - you can check the latest version [here](Started writing the Cryptarchia) .
```
Comment (alvatar) [inline content/nomos/updates/2024-01-15.md]: specifications are always Research

File: content/nomos/updates/2024-01-15.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+- Refactor consensus engine in preparation for adding a new consensus - [PR](https://github.com/logos-co/nomos-node/pull/556).
+- 
+
+## `data availability`
+
+### `research`
+
+- The [DA Layer Comparison table](https://www.notion.so/DA-Layer-Comparison-Table-5848811f0af042e2b24c10d3cea9b0a8) has been finished and is currently in review and update phase. For the
```
Comment (alvatar) [inline content/nomos/updates/2024-01-15.md]: Unfinished sentence

File: content/nomos/updates/2024-01-15.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+### `research`
+
+- The [DA Layer Comparison table](https://www.notion.so/DA-Layer-Comparison-Table-5848811f0af042e2b24c10d3cea9b0a8) has been finished and is currently in review and update phase. For the 
+- 
+
+### `development`
+
+- Merged a couple of small fixes for the nomos-chat app - more details [here](https://github.com/logos-co/nomos-node/pull/555).
+- [DA API](https://www.notion.so/DA-Api-specification-83000d9c2f3f45b586cd55c5c195cce5) specification has been added.
```
Comment (alvatar) [inline content/nomos/updates/2024-01-15.md]: Specs alwasys in research (regardless of who does it), to simplify

File: content/nomos/updates/2024-01-15.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+- [Block format specification](https://www.notion.so/Block-Format-Specification-10c4cfef8b8946b8a54344cf97b16938) has been added.
```
Comment (alvatar) [inline content/nomos/updates/2024-01-15.md]: Ditto

File: content/nomos/updates/2024-01-15.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+## `miscellaneous`
+
+- 3 interrelated topics that have the potential to create an interesting element of differentiation have been researched: turning Execution Zones into Plasma chains with ZK proofs, solutions for instant deposit/withdrawal, solutions for ZK-bridging with the Base Layer (basically the CL layer, but as minimal as possible) - [findings](https://www.notion.so/Plasma-ca4bfeb776e240859f954e8511103111).
```
Comment (alvatar) [inline content/nomos/updates/2024-01-15.md]: The link is only for the Plasma one, please add it there only

File: content/nomos/updates/2024-01-15.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+- Polygon Avail has also been researched - [findings](https://www.notion.so/Polygon-Avail-bc5bfbbf78c74b9aba6d4500d961ef03).
+- [Simulations working principle](https://www.notion.so/Simulations-working-principle-Carnot-paper-Appendix-c83740646adf4dd58e6b3212f7c1b666) (the Carnot paper Appendix) has been added.
+- Whitepaper feedback review in progress.
+- Carnot paper has also been reviewed.
+- Reviewed Ouroboros Praos, the focus was to understand the whole design and put a bit more attention at the design of the random beacon and some security assumptions. More on that in the [notes](https://www.notion.so/Ouroboros-Family-8cd7285459f640b088716905ab03474f)
```
Comment (alvatar) [inline content/nomos/updates/2024-01-15.md]: This goes in Cryptarchia

**Review by alvatar — CHANGES_REQUESTED** [review]

File: content/nomos/updates/2024-01-15.md (hunk 1 above)
Comment (Cofson) [reply content/nomos/updates/2024-01-15.md]: Removed. When the discussion is finished and a result is reached - I can add it in the next (or one of the next) updates.

File: content/nomos/updates/2024-01-15.md (hunk 2 above)
Comment (Cofson) [reply content/nomos/updates/2024-01-15.md]: Changed.

File: content/nomos/updates/2024-01-15.md (hunk 3 above)
Comment (Cofson) [reply content/nomos/updates/2024-01-15.md]: Moved into research.

File: content/nomos/updates/2024-01-15.md (hunk 4 above)
Comment (Cofson) [reply content/nomos/updates/2024-01-15.md]: Finished now - added the link to the gsheet.

File: content/nomos/updates/2024-01-15.md (hunk 5 above)
Comment (Cofson) [reply content/nomos/updates/2024-01-15.md]: Moved into research.

File: content/nomos/updates/2024-01-15.md (hunk 6 above)
Comment (Cofson) [reply content/nomos/updates/2024-01-15.md]: Moved into research.

File: content/nomos/updates/2024-01-15.md (hunk 7 above)
Comment (Cofson) [reply content/nomos/updates/2024-01-15.md]: Added it into brackets next to Plasma.

File: content/nomos/updates/2024-01-15.md (hunk 8 above)
Comment (Cofson) [reply content/nomos/updates/2024-01-15.md]: Moved.

**Review by alvatar — APPROVED** [review]

**Pull request merged**